server/src/core/import_resolver.rs

```diff
@@ -57,6 +57,9 @@ fn resolve_import_stmt_hook(alias: &Alias, from_symbol: &Option<Rc<RefCell<Symbo
  * Helper to manually import a symbol. Do not forget to use level instead of '.' in the from_stmt parameter.
  */
 pub fn manual_import(session: &mut SessionInfo, source_file_symbol: &Rc<RefCell<Symbol>>, from_stmt:Option<String>, name: &str, asname: Option<String>, level: u32, diagnostics: &mut Option<&mut Vec<Diagnostic>>) -> Vec<ImportResult> {
+    if source_file_symbol.borrow().path().as_vec().len() > 1 {
+        panic!("Importing from a multi-path namespace symbol is not supported in manual_import");
+    }
```

Comment:
As a reminder: this makes it possible that `source_file_symbol` is a namespace symbol (thus, with Multiple paths) in `resolve_from_stmt` and `resolve_import_stmt`